glossary.yml

```
@@ -1919,12 +1955,34 @@
       the relationships between values, and ways to access or manipulate those data, such as
       reading, altering, or writing.
 
+- slug: data_type
+  en:
+    term: "data type"
+    def: >
+      An attribute of data which defines what operations can be performed
+      and defines the range of values that can be stored in a data object
+      of a given type. Examples of common types include: boolean or logical,
+      binary, integer,  floating point numeric, character or string.     
+  ja:
+    term: "データ型"
+    def: >
+      演算を使用出来って、データー実体に入られるの設定範囲を
```

```suggestion
      使用可能な演算操作、およびデータオブジェクトに格納できる値の設定範囲を
```

---

Correct me if I'm mistaken but "演算操作" is specifically for numeric/mathematical operations and 操作 is more general and applies to string/character and other types as well?

```suggestion
      使用可能な操作、およびデータオブジェクトに格納できる値の設定範囲を
```